esmf-semantic-aspect-meta-model/src/main/resources/samm/characteristic/2.0.0/characteristic-shapes.ttl

```diff
@@ -783,3 +783,27 @@ samm-c:FixedPointConstraintShape
       sh:name "integer" ;
       sh:description "The number of integral digits for a fixed point number." ;
    ] .
+
+samm-c:TimeSeriesShape
+   a sh:NodeShape ;
+   sh:targetClass samm-c:TimeSeries ;
+   sh:property [
+      sh:path samm:dataType ;
+      sh:minCount 0 ;
+      sh:maxCount 1 ;
+      sh:name "dataType" ;
+      sh:description "This constraint for TimeSeries type and for checking extending this type from TimeSeriesEntity." ;
```

Comment:
This formulation is a little bit confusing and needs improvement.
For example "extending this type from TimeSeriesEntity" does not apply for the TimeSeriesShape, it's data type has to extend it.